src/gen_dragonfruit/gd_dev_apogee_sp215.cpp

```diff
@@ -21,7 +21,7 @@ int gd_dev_apogee_sp215_read(void){
         value += Wire.read();
     }
     Wire.endTransmission();
-    //value = (value*5000.00)/(0x7FFF);
+    value = (value*5000.00)/(0x7FFF);
```

Comment:
Is there a reason this is 0x7FFF? I'm not sure the context here. In this case, it might be good to write the conversion factor differently (if it's a conversion factor) or put comments to describe what it's doing.